packages/runtime/src/oneof.ts

```diff
@@ -49,16 +48,16 @@ export function isOneofGroup(any: any): any is UnknownOneofGroup {
  * Returns the value of the given field in a oneof group.
  */
 export function getOneofValue<T extends UnknownOneofGroup,
```

Comment:
With the new representation, this function doesn't provide much use anymore. We should deprecate it, and add a note that it is going to be removed in the next release.

Reply:
Sorry, do you mean that we should deprecate this function in the 3.0 release and remove it later or that we should deprecate this function in a 2.x release and remove it in the 3.0 release?

Reply:
Deprecated in 3.0 and remove later. It doesn't hurt to keep it around so users can migrate more easily.